apps/css-workshop/src/pages/tree.astro

```diff
@@ -385,11 +386,7 @@ import Layout from './_layout.astro';
     <ul class='iui-tree' role='tree'>
       <li class='iui-tree-item' role='treeitem' tabindex='0' aria-expanded='true'>
         <div class='iui-tree-node'>
-          <input
-            class='iui-checkbox iui-checkbox-visibility iui-tree-node-checkbox'
-            type='checkbox'
-            checked
-          />
+          <Checkbox_ variant='basic eyeball' checked />
```

Comment:
Looks like there is a type error for `variant` here and other similar places in this file.

Instead of re-using the variant prop, maybe you can consider adding some other prop for with/without wrapper. This will help make the `variant` prop accept the same values as the `variant` prop in `Checkbox.tsx`.

Reply:
adde a new `basic` prop to avoid over using variant